File: bulk/csharp/SqlInjection01.cs
// CWE-89: SQL Injection via string concatenation
using System.Data.SqlClient;
public class SqlInjection01 {
    public object GetUser(SqlConnection conn, string username) {
        var cmd = new SqlCommand("SELECT * FROM Users WHERE Username = '" + username + "'", conn);
        return cmd.ExecuteReader();
    }
}

File: bulk/java/SqlInjection01.java
// CWE-89: SQL Injection via Statement
import java.sql.*;
public class SqlInjection01 {
    public ResultSet getUser(Connection conn, String username) throws SQLException {
        Statement stmt = conn.createStatement();
        return stmt.executeQuery("SELECT * FROM users WHERE username = '" + username + "'");
    }
}
